java/sample-apps/okhttp/deploy/wrapper/layer_arm64.tf

```diff
@@ -1,14 +1,21 @@
+
 locals {
   sdk_layer_arns_arm64 = {
-    "ap-northeast-1" = "arn:aws:lambda:ap-northeast-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-18-0:1"
-    "ap-south-1"     = "arn:aws:lambda:ap-south-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-18-0:1"
-    "ap-southeast-1" = "arn:aws:lambda:ap-southeast-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-18-0:1"
-    "ap-southeast-2" = "arn:aws:lambda:ap-southeast-2:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-18-0:1"
-    "eu-central-1"   = "arn:aws:lambda:eu-central-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-18-0:1"
-    "eu-west-1"      = "arn:aws:lambda:eu-west-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-18-0:1"
-    "eu-west-2"      = "arn:aws:lambda:eu-west-2:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-18-0:1"
-    "us-east-1"      = "arn:aws:lambda:us-east-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-18-0:1"
-    "us-east-2"      = "arn:aws:lambda:us-east-2:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-18-0:1"
-    "us-west-2"      = "arn:aws:lambda:us-west-2:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-18-0:1"
+    "ap-northeast-1" = "arn:aws:lambda:ap-northeast-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-19-0:1"
+    "ap-northeast-2" = "arn:aws:lambda:ap-northeast-2:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-19-0:1"
+    "ap-south-1"     = "arn:aws:lambda:ap-south-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-19-0:1"
+    "ap-southeast-1" = "arn:aws:lambda:ap-southeast-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-19-0:1"
+    "ap-southeast-2" = "arn:aws:lambda:ap-southeast-2:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-19-0:1"
+    "ca-central-1"   = "arn:aws:lambda:ca-central-1:901920570463:layer:aws-otel-java-wrapper-arm64-ver-1-19-0:2"
```

Comment:
Why is this particular region on a different layer version than the rest?

Reply:
I have copied this from the `tf` layer generated in the [workflow](https://github.com/aws-observability/aws-otel-lambda/actions/runs/3425866372/jobs/5707088914#step:8:1), ca-central-1 region has published the layer version 2,
 The initial workflow has encountered an error and cancelled. [version 1 ](https://github.com/aws-observability/aws-otel-lambda/actions/runs/3425767316/jobs/5706875157#step:9:81)was copied/published (not a public layer yet) to prod at this time for this region. when retriggered then, the correct layer came out as version 2.